plugins/login/login.js

```diff
@@ -12,6 +14,51 @@ const whitelist = {
     stjohnjohnson: true,
     tkyi: true
 };
+const hashr = require('screwdriver-hashr');
+const iron = require('iron');
+const Model = require('screwdriver-models');
+
+/**
+ * [getUser description]
+ * @param  {[type]} config [description]
+ * @return {[type]}        [description]
+ */
+function getUser(config, callback) {
+    const User = new Model.User(config.datastore);
+
+    return User.get(config.id, callback);
+}
+
+/**
+ * [sealToken description]
+ * @param  {[type]}   config   [description]
+ * @param  {Function} callback [description]
+ * @return {[type]}            [description]
+ */
+function sealToken(config, user, callback) {
+    const User = new Model.User(config.datastore);
+    let userConfig;
+
+    iron.seal(config.token, config.password, iron.defaults, (error, sealed) => {
```

Comment:
You should move this into the User model, the sealing/unsealing of a token will probably be resuable :P